testsuite/libra-fuzzer/src/main.rs

```diff
@@ -35,7 +33,7 @@ enum Command {
         #[structopt(
             short = "n",
             long = "num-items",
-            default_value("&GENERATE_DEFAULT_ITEMS_STR")
+            default_value = &GENERATE_DEFAULT_ITEMS_STR
```

Comment:
Turns out this was just plain broken and was using the str "&GENERATE_DEFAULT_ITEMS_STR" instead of the lazily evaluated string "128". This was caught because the compiler started throwing warnings about that static variable being unused.